cppSettings.gradle

```diff
@@ -72,6 +72,8 @@ task clean(type: Delete) {
     delete buildDir
 }
 
+ext.toolChainPath = null
```

Comment:
This should be an if-not-set check, like https://github.com/wpilibsuite/allwpilib/blob/master/build.gradle#L8. We also need this option added to the README.